planet/exceptions.py

```diff
@@ -68,3 +68,7 @@ class InvalidIdentity(APIException):
 class RequestCancelled(Exception):
     '''Internal exception when a request is cancelled'''
     pass
+
+
+class OrderError(APIException):
```

Comment:
+1 to an exception that is specific to each client. I am curious why you chose Error here instead of Exception.

Reply:
`*Error` is the Python community consensus favorite. The standard library has `ValueError` not `ValueException`. Attrs and urllib3 are two of the most widely used non-standard libs and those use Error in the names: https://urllib3.readthedocs.io/en/latest/reference/urllib3.exceptions.html, https://github.com/python-attrs/attrs/blob/main/src/attr/exceptions.py. It's recommended in the Google Python style guide, too, which we kinda adhere to because of yapf, right?